tests/components/switch_as_x/test_init.py

```diff
@@ -36,7 +38,7 @@ async def test_config_entry_unregistered_uuid(
     assert len(hass.states.async_all()) == 0
 
 
-@pytest.mark.parametrize("target_domain", (Platform.LIGHT,))
+@pytest.mark.parametrize("target_domain", (Platform.LIGHT, Platform.SIREN))
```

Comment:
Shouldn't this be tested for cover?

Reply:
It can't as cover isn't a toggle entity, thus not have on/off states

Reply:
OK, we should rework the test so it's parametrized with the expected states of the `switch_as_x` entity in another PR.